src/controllers/ArticleController.js

```diff
@@ -113,11 +113,17 @@ export default class ArticleController {
    * @returns {object} Object representing the response returned
    */
   static async update(req, res) {
+    // if (req.article.userId === req.user.id) {
+    //   return res.status(status.UNAUTHORIZED).send({
+    //     errors: { author: 'Sorry, you are not the author of this article' }
+    //   });
+    // }
+
     let updateArticle = {};
     let message = '';
     if (Object.keys(req.body).length > 0) {
       updateArticle = {
```

Comment:
I think this update method may be refactored to be more dynamic, as I see here to update an article you have to provide all the fields, what if someone only wants to update the title? I think to avoid updating unnecessary fields you can only update what is needed

Reply:
@rwajon check, it's updated